docs/flow/tutorials/collecting-prometheus-metrics.md

```diff
@@ -0,0 +1,65 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/collecting-prometheus-metrics
+title: Collecting Prometheus metrics
+weight: 200
+---
+
+Grafana Agent is a telemetry collector and the primary goal is to get telemetry data from one location to another. The goal of this tutorial is to get a basic working example of Grafana Agent working in Flow mode.  
```

Comment:
Do we want to explicitly name the 'pipeline' concept here? I personally think it describes nicely what we're trying to achieve.

Reply:
After talking with robert yesterday about introducing people to prometheus/agent.  I think pipeline discussion will happen in a future tutorial. Conversely, the `chaining` article might be a good place.

Reply:
Context about our conversation yesterday:

I've been thinking about how we get feedback that our docs are tailored towards people who are already familiar with Prometheus, but the audience of people using Grafana Agent has been growing well beyond that user base. 

It seemed like a good idea for us to use these tutorials as a way to explain Prometheus concepts through Flow, teaching both at the same time, so I passed that suggestion on to Matt.